includes/vendor/composer/installed.json

```diff
@@ -2,28 +2,29 @@
     "packages": [
         {
             "name": "aura/sql",
-            "version": "3.1.0",
-            "version_normalized": "3.1.0.0",
+            "version": "6.0.0",
```

Comment:
I might have missed something, but are we using version 5 or version 6?

Reply:
The idea is to use version 5 as suggested by @srjlewis in https://github.com/auraphp/Aura.Sql/commit/92df92269e1961304d7c46ef0046d71b6f0f3fec#commitcomment-152736781. Version 5 supports PHP < 8.4

Reply:
Yes, that is clear 😊
But aren't the files in here coming from version 6?

Reply:
Yes, how would you like handle the changes in `includes/vendor/`? There would be around 80 changes such as:
- updating aura/sql
- updating modules due to the change in the PHP version
- removing old dependencies such as includes/vendor/symfony/polyfill-php72

Usually I would not include `includes/vendor` but use the composer files instead to generate this folder automatically.

Reply:
> Usually I would not include `includes/vendor`

Yes, I'd prefer too. But for history reasons, `includes/vendor` has to be part of the repo.
I'll do my best to find a correct resolution 😊